helm/promtail/templates/service.yaml

```diff
@@ -1,16 +1,18 @@
+{{- if .Values.giantswarm.labelMonitoring.enabled }}
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "promtail.fullname" $ }}
+  name: {{ .Values.giantswarm.labelMonitoring.name }}
   namespace: {{ $.Release.Namespace }}
   {{- with .annotations }}
   annotations:
     {{- toYaml . | nindent 4 }}
   {{- end }}
   labels:
-    {{- include "promtail.selectorLabels" . | nindent 8 }}
-    {{- with .Values.serviceLabels }}
-    {{- toYaml . | nindent 8 }}
+    {{- include "promtail.selectorLabels" . | nindent 4 }}
```

Comment:
Not sure, but should we keep `promtail.selectorLabels` here?
Won't it be a problem if we also have the servicemonitor enabled? Maybe it will scrape the "normal" service as well as this one then?

Reply:
Well, normally once we'll enable the serviceMonitor we won't need this headless service right ? So it should be disabled at that point and thus leaving the selector labels wouldn't be an issue.